pkg/controller/pod.go

```diff
@@ -102,28 +106,32 @@ func (r *ReconcilePod) createReplicas(ctx context.Context, pod *corev1.Pod) erro
 
 		// get all replica attachments for the given volume
 		replicaNodes, err := getNodesWithReplica(ctx, r, volume)
-		// if there already are replica attachments for the volume, let the replica reconciler handle replica creation and skip batch creation to avoid race between two controllers
-		if len(replicaNodes) > 0 {
-			continue
-		} else if err != nil {
+		if err != nil {
+			klog.Warning(fmt.Sprintf("Error getting replica azvolumes for pod %s and volume %s. Error: %v", podKey, volume, err))
 			return err
 		}
-
-		numCreated := 0
-		for _, node := range nodes {
-			if numCreated >= azVolume.Spec.MaxMountReplicaCount {
-				break
-			}
-			if err := createReplica(ctx, r, azVolume.Status.Detail.ResponseObject.VolumeID, node); err != nil {
-				return err
+		// if there already are replica attachments for the volume, let the replica reconciler handle replica creation and skip batch creation to avoid race between two controllers
+		if len(replicaNodes) > 0 {
```

Comment:
np: can we revert to `if { continue } ` format? I think it would help in terms of maintaining code indentation as we wouldn't need to put everything into `else` clause then.

Reply:
It might just be me but I really don't like to use goto statements in the code. I think it breaks the readability of the code. Do you think it is better approach?

Reply:
This also might just be me but I prefer to use continue or break when possible to avoid over usage of if-else clauses, I think it helps with the readability as it helps prevent indentation overflow and allows readers to easily identify where the code should skip or break iteration.

Reply:
I know the C#-style guidelines recommend using `continue` and `break` statements in loops to avoid unnecessary indentation. I haven't found a definitive Go style recommendation, but most Go code I've seen takes this approach as well. Regardless, we should be consistent in a given codebase and this one generally uses `continue` and `break` statements to reduce indentation.